### PR TITLE
Created portfolio body

### PR DESCRIPTION
**Pull request by anthonymr**
## Milestone II - Created works and about sections.

In this PR, I did the following things:

### General:
- Added works and language picture files

### Works and about  sections:
- Added basic HTML structure
- Added CSS style according to the Sigma first template.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -22,23 +22,160 @@
...
+                </li>
+            </ul>
+            <div class="about-myself_menu-item">
+                <h4>Frameworks</h4>
+                <img src="img/chevron-right.svg">
+            </div>
+            <div class="about-myself_menu-item">
+                <h4>Skills</h4>
+                <img src="img/chevron-right.svg">
+            </div>
```
Comment (Ugboaja-Uchechi) [inline index.html]: - [ ] As stated in the project requirements, kindly make sure you group the categories in the `About myself` section `Languages`, `Skills`, and `Framework` in a `ul` tag and each of them in an `li` tag 🙏

File: css/style.css [hunk 2]
```diff
@@ -145,3 +156,154 @@ button:active:enabled {
...
+.about-myself > .social_wrapper {
+  margin: 3px 0;
+}
+
+.about-myself h4 {
+  font-size: 20px;
+  font-weight: 500;
+  color: var(--black);
+  margin: 0 0 0 12px;
+}
```
Comment (Ugboaja-Uchechi) [inline css/style.css:269]: - [ ] Great job with your design :clap: Kindly make sure you add the icon beside the `language` text 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @anthonymr, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (anthonymr) [reply index.html]: Thanks, I misunderstood that requirement.

File: css/style.css (hunk 2 above)
Comment (anthonymr) [reply css/style.css:269]: Thanks for showing me. The comment bubbles in Figma had hidden the arrow.

**Review by OyePriscilla — APPROVED** [review]
Hi @anthonymr  👍 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Linters check passed ✔️ 
Good commit messages ✔️ 
GitHub flow observed 😄 
Excellent communication with Code Reviewer ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**